api/src/test/java/org/pmiops/workbench/workspaces/WorkspacesControllerTest.java

```diff
@@ -165,8 +163,6 @@
 
 @RunWith(SpringRunner.class)
 @DataJpaTest
-@Import(LiquibaseAutoConfiguration.class)
-@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
 @DirtiesContext(classMode = ClassMode.BEFORE_EACH_TEST_METHOD)
 @Transactional(propagation = Propagation.NOT_SUPPORTED)
```

Comment:
Is this `@Transactional` not needed any other places?

Reply:
So any test that has `@Transactional(propagation = Propagation.NOT_SUPPORTED)` i left alone since they are overriding the behavior of Transactional. So I only removed `@Transactional` since it's inherited from `@DataJpaTest`